src/scheduling.jl

```diff
@@ -6,6 +6,9 @@ if Preferences.@load_preference("distributed-package") == "DistributedNext"
 else
     using Distributed
 end
+import NVTX
+
+nvtx_color = Colors.distinguishable_colors(32)
```

Comment:
```suggestion
import NVTX
import Colors

nvtx_color = Colors.distinguishable_colors(32)
```
The tests are failing because import is missing